content/en-us/tutorials/fundamentals/coding-4/repeating-code-with-while-loops.md

```diff
@@ -121,16 +121,16 @@ With the while loop created, instructions can be added to change the part's colo
 
 ### Making the Script Wait
 
-If you add a second line of color changing code right now, it would change the brick's color so fast you might not even see the first color go by. To make the script wait before running the next line of code, use a `wait()` function.
+If you add a second line of color changing code right now, it would change the brick's color so fast you might not even see the first color go by. To make the script wait before running the next line of code, use a `task.wait()` function.
```

Comment:
Adding `Libary.` makes the code automatically link to the [reference page](https://create.roblox.com/docs/reference/engine/libraries/task#wait) (see [STYLE.md](https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links)):

```suggestion
If you add a second line of color changing code right now, it would change the brick's color so fast you might not even see the first color go by. To make the script wait before running the next line of code, use a `Library.task.wait()` function.
```